src/content/1.7/basics/keeping-up-to-date/migration.md

```diff
@@ -566,7 +566,7 @@ regenerate all the missing sizes.
 
 ### Attachments
 
-Like the images, these folders need to be copied on the new server:
+Like the images, these folders needs to be copied on the new server:
```

Comment:
Same as https://github.com/PrestaShop/docs/pull/1018#discussion_r640365443

Reply:
You're right